docs/root/configuration/statistics.rst

```diff
@@ -23,6 +23,7 @@ Server related statistics are rooted at *server.* with following statistics:
   version, Gauge, Integer represented version number based on SCM revision
   days_until_first_cert_expiring, Gauge, Number of days until the next certificate being managed will expire
   hot_restart_epoch, Gauge, Current hot restart epoch
+  initialization_time_ms, Histogram, Total time taken for Envoy initialization in milliseconds
```

Comment:
Think it might be worth adding a bit more details here, for folks less familiar with Envoy internals?

Total time taken for Envoy initialization in milliseconds.  This is the time from server start-up until the worker threads are ready to accept new connections

?

Reply:
Yes. Added